include/RAJA/policy/cuda/forall.hpp

```diff
@@ -68,7 +68,7 @@ namespace impl
  ******************************************************************************
  */
 RAJA_INLINE
-dim3 getGridDim(size_t len, dim3 blockDim)
+unsigned getGridDim(size_t len, dim3 blockDim)
```

Comment:
@ajkunen is returning 'unsigned' when gridSize is 'size_t' potentially problematic?